jaseci_serv/jaseci_serv/svc/config.py

```diff
@@ -0,0 +1,7 @@
+import sys
+
+KUBERNETES_CONFIG = {"in_cluster": False, "config": None}
+
+META_CONFIG = {"automation": False, "monitoring": True, "kubernetes": KUBERNETES_CONFIG}
```

Comment:
What is the relationship between the config in jaseci_serv and the config in jaseci_cor? Can we avoid having two very similar `config.py` and `META_CONFIG` in the repo?

Reply:
Removed